.github/workflows/appimage.yml

```
@@ -140,3 +140,53 @@ jobs:
           -H "X-Bintray-Override:1"
           -H "X-Bintray-Publish:1"
           https://api.bintray.com/content/cryptomator/cryptomator/${GITHUB_REF##*/}/cryptomator-${APPIMG_VERSION}-x86_64.AppImage.zsync
+
+  publish-github:
+    name: Publish on Github 
+    runs-on: ubuntu-latest
+    if: startsWith(github.ref, 'refs/tags/') # only when tagged
+    needs: build-appimage
+    steps:
+      - name: Get version
+        id: getversion
+        run: echo "APPIMG_VERSION=${{ needs.build-appimage.outputs.appimage-version }}" >> $GITHUB_ENV
+      - name: Download AppImage
+        uses: actions/download-artifact@v1
+        with:
+          name: cryptomator-${{ env.APPIMG_VERSION }}-x86_64.AppImage
+          path: .
+      - name: Download AppImage.zsync
+        uses: actions/download-artifact@v1
+        with:
+          name: cryptomator-${{ env.APPIMG_VERSION }}-x86_64.AppImage.zsync
+          path: .
+      - name: Create release on Github
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

```suggestion
          GITHUB_TOKEN: ${{ secrets.CRYPTOBOT_RELEASE_TOKEN }} # release as "cryptobot"
```

If the release is not created by "GitHub Actions" but rather by "cryptobot", it can trigger subsequent workflows. The assets can still be uploaded with the normal GITHUB_TOKEN, though.